perf/benchmark/runner/runner.py

```diff
@@ -212,10 +212,13 @@ def run(self, conn, qps, size, duration):
 
 
 PERFCMD = "/usr/lib/linux-tools/4.4.0-131-generic/perf"
+FLAMESH = "flame.sh"
 PERFSH = "get_perfdata.sh"
 PERFWD = "/etc/istio/proxy/"
 WD = os.getcwd()
+LOCAL_FLAMEPATH = os.path.join(WD, "../flame", FLAMESH)
```

Comment:
create a new variable for flame directory path, then you don't need to join all these together each time